script/create_zip.sh

```diff
@@ -37,7 +37,7 @@ ext_name=$2
 ext_version=$3
 cd $ext_dir
 ext_dir=`pwd`
-cd $curr_dir
+cd "$curr_dir"
```

Comment:
I find this `pwd` thing pretty sloppy. we can use relative paths to the build script by using `readonly SCRIPT_DIR=$(dirname $0)` and find our way from there